### PR TITLE
Added support for add custom hue values when defining theme palettes

### PR DESCRIPTION
**Pull request by RasmusTG**
Updated packages version
- Angular 1.5.8
- Angular-Material 1.1.1

### CODE REVIEW

**Comment by RasmusTG** [comment]
I signed it!

File: lib/builders/ThemeBuilder.ts [hunk 1]
```diff
@@ -115,23 +115,35 @@ interface MdThemingProvider {
...
+  accentPaletteHues?: IThemeHues;
   warnPalette?: string;
+  warnPaletteHues?: IThemeHues;
   backgroundPalette?: string;
+  backgroundPaletteHues?: IThemeHues;
   dark?: boolean;
 }
 
+/** Angular Material Theme hue registry */
+export interface IThemeHues {
```
Comment (DevVersion) [inline lib/builders/ThemeBuilder.ts]: Can we rename this to `MdThemeHues` and update the comment above to `Angular Material Hue definitions`

**Comment by DevVersion** [comment]
Also while you at it, would be good if you rebase the PR on top of the latest changes on `master`.

File: lib/builders/ThemeBuilder.ts (hunk 1 above)
Comment (RasmusTG) [inline lib/builders/ThemeBuilder.ts]: Yes, of cause.

If you also have any suggestion or how it should be done in the commandline, I will do this also.

I will also do documentation.

// Rasmus

Comment (DevVersion) [inline lib/builders/ThemeBuilder.ts]: Thank you very much! Yeah I don't think we should support those things in the Command Line, so we can leave that.

Comment (DevVersion) [inline lib/builders/ThemeBuilder.ts]: @RasmusTG Hey any updates on this PR? Would really like to get yours in.

**Pull request closed without merging**